lib/init.coffee

```diff
@@ -32,6 +35,11 @@ module.exports =
       type: 'string'
       default: 'luacheck'
       description: 'The executable path to luacheck.'
+    standard:
+      type: 'array'
+      default: []
+      description: "List of comma separated standards.
+        eg. `min, busted`"
```

Comment:
Was really close to 80 characters if I remember right... I may have changed the string before committing though so I don't remember